configuration/postgres/postgres.go

```diff
@@ -162,42 +163,40 @@ func (p *ConfigurationStore) Get(ctx context.Context, req *configuration.GetRequ
 }
 
 func (p *ConfigurationStore) Subscribe(ctx context.Context, req *configuration.SubscribeRequest, handler configuration.UpdateHandler) (string, error) {
-	var pgNotifyChannels []string
+	pgNotifyChannel := ""
 	for k, v := range req.Metadata {
-		if res := strings.EqualFold(pgNotifyChannelKey, k) && !slices.Contains(pgNotifyChannels, v); res {
-			pgNotifyChannels = append(pgNotifyChannels, v) // append unique channel names only
+		if strings.EqualFold(pgNotifyChannelKey, k) {
```

Comment:
Since these keys are all ASCII-only (or anyways we don't care if they're not ASCII), please do:

```suggestion
		if strings.ToLower(k) == pgNotifyChannelKey {
```

The linter will complain so you may need to add `//nolint:gocritic`

> strings.EqualFold is useful when the strings contain non-ASCII chars, but in this case is just slower.